src/lib/core/redux/slices/rtcConnection/actions.ts

```diff
@@ -10,3 +14,46 @@ export const rtcEvents = {
     rtcManagerDestroyed: createRtcEventAction<void>("rtcManagerDestroyed"),
     streamAdded: createRtcEventAction<RtcStreamAddedPayload>("streamAdded"),
 };
+
+type RtcAnalyticsCustomEvent = {
+    actionType: string;
```

Comment:
see next comments of why. 
```suggestion
    action: Matcher<UnknownAction>;
```